tensorboard/webapp/plugins/npmi/util/coordinate_data_test.ts

```diff
@@ -0,0 +1,128 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {AnnotationDataListing} from '../store/npmi_types';
+import {coordinateData} from './coordinate_data';
+
+describe('coordinate data utils', () => {
+  it('creates violin data containing only selected metric and active runs', () => {
```

Comment:
add a case when no runs are active, AnnotationDataListing is empty, and nothing matches the active metrics.